src/svc_auth_gss.c

```diff
@@ -628,15 +653,22 @@ _svcauth_gss(struct svc_req *req, bool *no_dispatch)
 		 */
 		unref_svc_rpc_gss_data(gd);
 		req->rq_auth = &svc_auth_none;
+		goto gc_free;
 
 		break;
 
 	default:
-		svcauth_gss_return(AUTH_REJECTEDCRED);
+		rc = AUTH_REJECTEDCRED;
 		break;
 	}
-
-	svcauth_gss_return(AUTH_OK);
+gd_free:
```

Comment:
Can we rename one of these?  Having them be one letter different makes them hard to track, and it's going to result in someone using the wrong one.

Reply:
Renamed gc_free to cred_free label.